Sources/SystemExtras/Syscalls.swift

```diff
@@ -47,6 +47,16 @@ internal func system_fcntl(_ fd: Int32, _ cmd: Int32) -> CInt {
   return fcntl(fd, cmd)
 }
 
+// fsync
+internal func system_fsync(_ fd: Int32) -> CInt {
+  return fsync(fd)
+}
+
+// fdatasync
+internal func system_fdatasync(_ fd: Int32) -> CInt {
+  return fdatasync(fd)
```

Comment:
On macOS, it seems that `fdatasync` can't be found.

```
/Users/runner/work/WasmKit/WasmKit/Sources/SystemExtras/Syscalls.swift:57:10: error: cannot find 'fdatasync' in scope
  return fdatasync(fd)
         ^~~~~~~~~
```

Reply:
fixed. https://github.com/swiftwasm/WasmKit/compare/1125ef011a9e4df3e35c90a4774184a578065a41..778ecd17916d79fdf021487f32c1a198808eda95